src/main/java/baritone/pathing/movement/MovementHelper.java

```diff
@@ -659,6 +662,57 @@ static void moveTowards(IPlayerContext ctx, MovementState state, BlockPos pos) {
         )).setInput(Input.MOVE_FORWARD, true);
     }
 
+    static void moveTowardsWithoutRotation(IPlayerContext ctx, MovementState state, BlockPos dest) {
+        float ax = Mth.sin(ctx.playerRotations().getYaw() * DEG_TO_RAD_F);
+        float az = Mth.cos(ctx.playerRotations().getYaw() * DEG_TO_RAD_F);
+        Rotation blockRotation = RotationUtils.calcRotationFromVec3d(ctx.playerHead(),
+                VecUtils.getBlockPosCenter(dest),
+                ctx.playerRotations());
+        int selection = getSelection(blockRotation, ax, az);
+        switch (selection) {
+            case 0 -> state.setInput(Input.MOVE_FORWARD, true);
+            case 1 -> state.setInput(Input.MOVE_BACK, true);
+            case 2 -> state.setInput(Input.MOVE_LEFT, true);
+            case 3 -> state.setInput(Input.MOVE_RIGHT, true);
+            case 4 -> state.setInput(Input.MOVE_FORWARD, true).setInput(Input.MOVE_LEFT, true);
+            case 5 -> state.setInput(Input.MOVE_FORWARD, true).setInput(Input.MOVE_RIGHT, true);
+            case 6 -> state.setInput(Input.MOVE_BACK, true).setInput(Input.MOVE_LEFT, true);
+            case 7 -> state.setInput(Input.MOVE_BACK, true).setInput(Input.MOVE_RIGHT, true);
+            default -> {}
+        }
+    }
+
+    private static int getSelection(Rotation blockRotation, float ax, float az) {
+        float targetAx = Mth.sin(blockRotation.getYaw() * DEG_TO_RAD_F);
+        float targetAz = Mth.cos(blockRotation.getYaw() * DEG_TO_RAD_F);
+        Vec2[] options = getOptions(ax, az);
+        int selection = -1;
+        float closestX = 100000;
+        float closestZ = 100000;
+        for (int i = 0; i < options.length; i++) {
+            if (Mth.abs(targetAx - options[i].x) + Mth.abs(targetAz - options[i].y) < closestX + closestZ) {
```

Comment:
This usage of x and y is unintuitive, maybe the Option should have as fields motionX and motionZ so that using `y` for `z` isn't happening anymore